cmd/tk/env.go

```diff
@@ -50,7 +51,7 @@ func envSetCmd() *cobra.Command {
 	envSettingsFlags(&tmp, cmd.Flags())
 
 	name := cmd.Flags().String("name", "", "")
-	cmd.Flags().MarkHidden("name")
+	_ = cmd.Flags().MarkHidden("name")
```

Comment:
nit: can we remove the redundant assign here?

Reply:
it returns an error, the linter is unhappy about it. This was intended to singal that we don't care about the error

Reply:
Got it 👍